### PR TITLE
Arranged the readme file. Added some changes in readme file

### PR DESCRIPTION
**Pull request by ghost**
Pull Request for Issue # .

### CODE REVIEW

File: build/build.js [hunk 1]
```diff
@@ -17,6 +17,7 @@
  * node build.js --versioning       will update all the joomla.assets.json files providing accurate versions for stylesheets and scripts.
  */
 
+
```
Comment (dgrammatiko) [inline build/build.js:20]: Revert this change

**Review by dgrammatiko — CHANGES_REQUESTED** [review]

**Comment by brianteeman** [comment]
This is a pointless change and should be closed

**Comment by ghost** [comment]
Sorry for the inconvenience.
I am a student and
I just learned github and wanted to contribute, but didn't know where to
start.

**Comment by richard67** [comment]
It seems the user has deleted their GitHub account and removed the fork repository.

Closing for the reasons stated in previous comments.

**Comment by PhilETaylor** [comment]
Hacktoberfest is just a few days away - be prepared for a lot more of these!

**Comment by brianteeman** [comment]
I checked my calendar when I saw it

**Comment by richard67** [comment]
I know. If the author hadn't deleted their account I would have written some (hopefully helpful) instructions on how to play around with Git/GitHub in their own fork first for getting familiar. But now as we have no channel back to the user, we only can hope we haven't discouraged him/her from further contributions.
